ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -162,16 +162,16 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	// Check if cluster is specified
 	if ecsParams.Cluster == "" {
-		return fmt.Errorf("Please configure a cluster using the configure command.")
+		return fmt.Errorf("Please configure a cluster using the configure command or with '--%s' flag", command.ClusterFlag)
```

Comment:
@yinshiua That comment applies to programmer-facing errors (or to something that may appear in logs), not to things that may be displayed to a user.